src/Microsoft.AspNetCore.Rewrite/Internal/UrlRewrite/UrlRewriteRuleBuilder.cs

```diff
@@ -53,10 +53,9 @@ public UrlRewriteRule Build()
                     _action = new RedirectAction(statusCode, url, appendQueryString);
                     break;
                 case ActionType.AbortRequest:
-                    throw new NotImplementedException("Abort Requests are not supported");
+                    throw new NotImplementedException("Abort Requests are not implemented");
```

Comment:
Ok, so just to make sure we are consistent: NIE = didn't get around to it (because 12 weeks isn't that long), NSE = we won't be doing this, right?